appservice/src/deploy/waitForDeploymentToComplete.ts

```diff
@@ -27,7 +27,7 @@ export async function waitForDeploymentToComplete(client: SiteClient, kuduClient
         [deployment, permanentId, initialStartTime] = await tryGetLatestDeployment(kuduClient, permanentId, initialStartTime, expectedId);
         if ((deployment === undefined || !deployment.id)) {
             if (expectedId && Date.now() < maxTimeToWaitForExpectedId) {
-                ext.outputChannel.appendLine(formatDeployLog(client, localize('waitingForBuild', 'Waiting for build to trigger...')));
+                ext.outputChannel.appendLine(formatDeployLog(client, localize('waitingForBuild', 'Starting deployment...')));
```

Comment:
So I didn't think about the fact that this message is logged multiple times. In that case, "Waiting" makes more sense, but I don't think I like the "Waiting" messages.

I guess I'd prefer a message that was _truly_ more consistent with zipDeploy in the sense that it's only displayed once